src/styles/stars.css

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+}
+    
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+  
+.stars .stars__star:nth-child(-n+4) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
Don't hardcore the exact amount of active stars. Add active stars according to the modifier